### PR TITLE
Fix a few minor typos or oddly worded sentences in SDK examples

### PR DESCRIPTION
**Pull request by DFEvans**

### CODE REVIEW

File: notebooks/search_and_download.ipynb [hunk 1]
```diff
@@ -8,19 +8,21 @@
...
+    "Python 3.8 or higher is required for this package to work. To install the package from PyPI run: `pip install satellitevu`"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If the package can't be installed straight off the bat, you may need to install any extra dependencies our sdk needs first. The most common is having to install the appdirs pachage (`pip install appdirs`)."
+    "If the package can't be installed straight off the bat, you may need to install any extra dependencies our SDK needs first. The most common is having to install the appdirs package (`pip install appdirs`)."
```
Comment (DFEvans) [inline notebooks/search_and_download.ipynb:25]: This section strikes me as a little odd. Under what circumstances would we expect that Pip would either:
a) Fail to _install_ a package due to a missing runtime dependency?
b) Fail to install a dependency that's clearly indicated in pyproject.toml (and, I assume, is therefore going in the wheel)?

File: notebooks/search_and_download.ipynb [hunk 2]
```diff
@@ -39,10 +41,11 @@
...
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "These will be used to authenticate your access to the API. You can either set them as environment variables in the shell or in your Python script e.g.\n",
+    "These will be used to authenticate your access to the API. You can either set them as environment variables in the shell, or in your Python script - several methods are illustrated below.\n",
     "\n",
     "**Important: When your client is initially created, you will only have access to the Archive Search. To be able to submit and download orders, please ask one of the Platform team to authorize you for performing these actions.**"
```
Comment (DFEvans) [inline notebooks/search_and_download.ipynb:50]: Is this package intended for use by external consumers? If so, do they know how to "ask one of the Platform team"?

**Comment by DFEvans** [comment]
Couple of extra thoughts in comments in the code

File: notebooks/search_and_download.ipynb (hunk 1 above)
Comment (zhelinisivanesan) [reply notebooks/search_and_download.ipynb:25]: This appears to be outdated now and should be removed - previously we probably didn't have `appdirs` in the `pyproject.toml` (or another reason which I can't remember) hence that line but now that package` should be installed as a dependency.

File: notebooks/search_and_download.ipynb (hunk 2 above)
Comment (zhelinisivanesan) [reply notebooks/search_and_download.ipynb:50]: Good spot, I wrote the notebook for internal use but as it's a public repo, anybody should be able to use it as an example. The best thing is to possibly insert the [Contact Us](https://www.satellitevu.com/home/#contact-us) Link on our webpage but I'm not sure whether will people will want to reach the developers directly @c-wygoda ?

**Comment by JamesOConnor** [comment]
Closing this, as it's been hanging a while and we can do a review of the notebook all together, as well as include new functionality in the next weeks

**Pull request closed without merging**